src/System.Linq/src/System/Linq/DebugView.cs

```diff
@@ -97,4 +97,34 @@ public object[] Items
         [DebuggerBrowsable(DebuggerBrowsableState.Never)]
         private IEnumerable _enumerable;
     }
+
+    internal sealed class GroupingDebuggerProxy<TKey, TElement>
```

Comment:
@OmarTawfik It seems kind of misleading to prefix them with `SystemCore_` because Linq gets its own System.Linq.dll assembly on .NET Core, it's no longer in System.Core.dll.